Makefile

```diff
@@ -0,0 +1,33 @@
+.DEFAULT_GOAL = help
+
+DEV = docker-compose --file docker-compose.dev.yml
+
+help: ## Display this help text
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+install: ## Install dependencies locally
+	npm install
+
+build: ## Build the container
+	${DEV} build --pull
```

Comment:
IIRC `--pull` or `pull` in general has some problems with tagged images as it may try to pull something that should be built locally (and hence doesn't exist yet)? Will try to reproduce

Reply:
Can easily remove, as using precise versions _shouldn’t_ see them change.